cmd/nri-kubernetes/main.go

```diff
@@ -264,7 +264,15 @@ func buildClients(c *config.Config) (*clusterClients, error) {
 	}, nil
 }
 
-func createIntegrationWithHTTPSink(config *config.Config) (*integration.Integration, error) {
+func createIntegrationWithHTTPSink(config *config.Config, logger *log.Logger) (*integration.Integration, error) {
+	endpoint := net.JoinHostPort(sink.DefaultAgentForwarderhost, strconv.Itoa(config.Sink.HTTP.Port))
+
+	logger.Info("Waiting for agent container to be ready...")
+	err := sink.WaitForEndpoint(fmt.Sprintf("http://%s%s", endpoint, sink.DefaultAgentReadyPath), 2*time.Minute)
```

Comment:
is 2m an empiric value? could we reduced to 1m? I just have in mind that using a wrong license key could pop up an error faster

Reply:
Reduced to 1m, plus it's now configurable :)